.gitignore

```diff
@@ -31,6 +31,7 @@ gover.coverprofile
 profile.coverprofile
 
 debug
+__debug_bin
```

Comment:
Each one of your PR adds things to gitignore!

Reply:
well, you have commit something, right?